src/tests/ftest/get_remote_files.sh

```diff
@@ -0,0 +1,369 @@
+    files.
+
+ Options:
+      -z        Compress files
+      -t        Threshold value to determine classification of big logs
+      -a        Remote path to scp files to i.e. server-A:/path/to/file
+      -f        Local path to files for archiving/compressing/size checking
+      -c        Enable running cart_logtest.py
+      -x        Enable performing compression when running on VM system
+      -v        Display commands being executed
+      -h        Display this help and exit
+"
+}
+
+#######################################
+# CONSTANT variables.
+#######################################
+# shellcheck disable=SC2046,SC2086
+CART_LOGTEST_PATH="$(dirname $(readlink -f ${0}))/cart/cart_logtest.py"
+
+#######################################
+# Check if the files exist on this host.
+# Arguments:
+#   $1: local logs to process
+# Returns:
+#   0: Files exist or no matches were found
+#   1: Missing the log files argument
+#######################################
+check_files_input() {
+    if [ -n "${1}" ]; then
+        # shellcheck disable=SC2086
+        files=$(ls -d ${1} 2> /dev/null)
+        files_rc=$?
+        if [ -n "${files}" ] && [ "${files_rc}" -eq 0 ]; then
+            echo "Files found that match ${1}."
+        else
+            echo "No files matched ${1}. Nothing to do."
+            exit 0
+        fi
+    else
+        echo "Please specify -f option."
+        exit 1
+    fi
+}
+
+#######################################
+# Convert human readable size values to respective byte value.
+# Arguments:
+#   size: size value to convert to bytes. i.e. 5mb, 40gb, 6k, 10g
+# Returns:
+#   converted byte value.
+#######################################
+human_to_bytes() {
+    h_size=${1,,}
+
+    declare -A units=(
+        [b]=1
+        [kb]=$(python -c "print (2**10)")
+        [k]=$(python -c "print (2**10)")
+        [mb]=$(python -c "print (2**20)")
+        [m]=$(python -c "print (2**20)")
+        [gb]=$(python -c "print (2**30)")
+        [g]=$(python -c "print (2**30)")
+    )
+    if [[ "${h_size,,}" =~ ([0-9.]+)([kbmgt]+) ]]; then
+        val="${BASH_REMATCH[1]}"
+        unit="${BASH_REMATCH[2]}"
+        (( val = val*${units[${unit}]} ))
+        echo "${val}"
+    else
+        echo "Invalid value provided: ${1}"
+        return 1
+    fi
+}
+
+#######################################
+# List and tag files with a 'Y' or 'N' to classify
+# big files based on the threshold provided.
+# Arguments:
+#   logs_dir: path to local logs
+#   threshold: value to use as threshold to classify big files
+# Returns:
+#   stdout output listing the files prepended with a 'Y' or 'N' tag
+#######################################
+list_tag_files() {
+    # shellcheck disable=SC2016,SC1004,SC2086
+    awk_cmd='{ \
+    if (th <= $1){ \
+        print "  Y:",$1,$2} \
+    else { \
+        print "  N:",$1,$2}\
+    }'
+
+    # Convert to bytes and run
+    bts=$(human_to_bytes "${2}")
+    # shellcheck disable=SC2086
```

Comment:
Why is this being ignored instead of fixed?

Reply:
Had to disable this shellcheck error because I need it to use the argument as a glob, but shellcheck is asking to include double quotes to ${1} arg, but this creates an issue as it will take that argument as a string literal and `du` will give a "No such file or directory" error.
```
amjustin@boro-10:~ $ du -ab -d 1 "/home/amjustin/*.sh"
du: cannot access ‘/home/amjustin/*.sh’: No such file or directory
```